src/eko/scale_variations/expanded.py

```diff
@@ -52,11 +52,12 @@ def variation_as2(gamma, L, beta0, g0e2):
         variation at |NNLO|
     """
     return -gamma[1] * L + 1.0 / 2.0 * (beta0 * gamma[0] + g0e2) * L**2
+    # changing alpha will change the sign of beta*gamma0
```

Comment:
This comment should be expanded (like this is only understandable if you know all the background, which we will forget ourselves in a while), and possibly included in the docstring with a suitable reference.

After all, it was the main source of problems.